build.yaml

```diff
@@ -289,19 +307,37 @@ common_cli:
     ssh-add \$HOME/.ssh/id_rsa
 
   run_acceptance_aws_tests: &run_acceptance_aws_tests |
-    # we gather the return code, but make sure we come out of these tests with '0'
+    # we gather the return code but make sure we come out of these tests with 0
     # we store that code into RC and pass it to the end of the job execution.
-    parse_logs \$venv/bin/python admin/run-acceptance-tests --distribution \${DISTRIBUTION_NAME} --provider aws --dataset-backend aws --branch \${TRIGGERED_BRANCH} --build-server  http://\$(wget -qO- http://instance-data/latest/meta-data/public-ipv4)  --config-file /tmp/acceptance.yaml \${ACCEPTANCE_TEST_MODULE} ; RC=\$?
+    parse_logs \$venv/bin/python admin/run-acceptance-tests \
+    --distribution \${DISTRIBUTION_NAME} \
+    --provider aws --dataset-backend aws --branch \${TRIGGERED_BRANCH} \
+    --build-server  \
```

Comment:
it may change as we dissect the run-acceptance-tests flocker-provision code, but that's a story for another day. needs some thought